R/fit.R

```diff
@@ -145,9 +168,8 @@ refit_multiple_optimizers <- function(fit,
     FUN = fit_single_optimizer,
     control = controls,
     MoreArgs = list(
-      formula = old_formula,
-      data = old_data,
-      weights = old_weights,
+      tmb_data = fit$tmb_data,
+      formula_parts = fit$formula_parts,
       reml = fit$reml
```

Comment:
reml can be removed as tmb_data already has it

Reply:
but now just `weights` is removed?

Reply:
yes; `reml` is already part of `tmb_data`